src/vs/workbench/contrib/terminalContrib/suggest/test/browser/terminalCompletionModel.test.ts

```diff
@@ -253,4 +253,43 @@ suite('TerminalCompletionModel', function () {
 			});
 		});
 	});
+
+	suite('lsp priority sorting', () => {
```

Comment:
since there will be other languages, should we create a suite within this suite called python that contains these?

Reply:
totally! just did them here: [473b005](https://github.com/microsoft/vscode/pull/243528/commits/473b0051949b786d4ec3070685227e73d79a803e)